apps/opencs/view/world/tablebottombox.cpp

```diff
@@ -99,7 +99,7 @@ CSVWorld::TableBottomBox::TableBottomBox (const CreatorFactoryBase& creatorFacto
 
     mStatusBar->addWidget (mStatus);
 
-    mLayout->addWidget (mStatusBar);
+    //mLayout->addWidget (mStatusBar);
```

Comment:
Reminder to self - this is unrelated, and shouldn't have made to this PR. It is however required currently to prevent crashing of XFCE, while using OpenMW-CS in on Manjaro.

Reply:
I feel your pain, we're having issues with Qt5 at my work as well... crashing when application is closing down, somewhere deep in a Qt5 deconstructor for any release of Qt5 >= 5.9, everything before that worked fine.